radiospectra/spectrogram2/spectrogram.py

```diff
@@ -0,0 +1,358 @@
+import struct
+from pathlib import Path
+from datetime import datetime
+
+import cdflib
+import matplotlib.dates as mdates
+import numpy as np
+import pandas as pd
+from matplotlib import pyplot as plt
+from matplotlib.image import NonUniformImage
+
+import astropy.units as u
+from astropy.time import Time
+from astropy.visualization import quantity_support
+from sunpy.io import fits
+from sunpy.net import attrs as a
+from sunpy.time import parse_time
+from sunpy.util.datatype_factory_base import BasicRegistrationFactory
+
+quantity_support()
```

Comment:
What is this?

Reply:
I guess it doesn't need to be here but the time and frequency are astropy quantities, could move inside the plot calls?